Jenkinsfile-core-dev

```diff
@@ -20,9 +20,19 @@ pipeline {
         AWX_TEMPLATE = 37
         HOST = 'play.dhis2.org'
         INSTANCE_NAME = "${env.GIT_BRANCH == 'master' ? 'dev' : env.GIT_BRANCH + 'dev'}"
+        GITHUB_TOKEN = credentials('github-token')
     }
 
     stages {
+        stage ('Patch') {
+            steps {
+                echo 'Patching DHIS2 ...'
+                sh 'curl "https://raw.githubusercontent.com/radnov/dhis2-server-setup/master/ci/scripts/dhis2-cve-patcher.sh" -O'
```

Comment:
The script is available [here](https://github.com/dhis2/dhis2-server-setup/blob/master/ci/scripts/dhis2-cve-patcher.sh) too. Do you still want to use the fork?

Reply:
Nope, that's a mistake on my part - I was using my fork while testing, but here we should have the "upstream" `dhis2/dhis2-server-setup` repo 👍